### DISCUSSION TITLE
SteamGridDB artwork location on Deck - Where is it?

### DISCUSSION THREAD

**Discussion by Frenkone** [SteamDeckHomebrew/decky-loader#637] [Q&A]
Hi there,
I was forced to resintall the OS on my Deck so I lost all my covers downloaded using SteamGridDB on Decky Plugin.
Please note that I still have all my games installed and stuff (the system was not wiped out) so I should have those covers downloaded on my Deck in a not known folder...
I'd like to find it in order to to be obliged to download all over again.

Do you know where those covers are stored when downloaded using that plugin? 
Thanks so much.

**Comment by TrainDoctor** [comment]
Please seek out the official git repo for SteamGridDB rather than making a discussion in our repo. Thanks.
